R/parameters.R

```diff
@@ -17,18 +18,30 @@ sircovid_parameters_shared <- function(start_date, region,
     stop("'beta_type' must be 'piecewise-linear' or 'piecewise-constant'")
   }
 
-
   population <- population %||% sircovid_population(region)
 
+  if (is.null(initial_seed_pattern)) {
+    initial_seed_pattern <- 1.0
+  }
+
+  start_step <- start_date * steps_per_day
+  initial_seed_step <- floor(start_step)
+  initial_seed_value <-
+    seed_over_steps(start_step, initial_seed_pattern) * initial_seed_size
+
+  ## TODO: initial_step can come out here once lancelot updated, no
```

Comment:
is this now resolved?

Reply:
I decided to use the initial_step here and left it in - have removed the TODO comment